src/auth.resource.tsx

```diff
@@ -0,0 +1,16 @@
+import { getCurrentUser, openmrsFetch } from "@openmrs/esm-api";
+
+export default {
+  getLoggedInUser,
+  logoutUser
+};
+
+function getLoggedInUser() {
```

Comment:
is this function worth it? Maybe we just use `getCurrentUser` directly inside of the component?

Reply:
Agreed